### PR TITLE
[lavender-flow] use flow match euler scheduler

### PR DESCRIPTION
**Pull request by yiyixuxu**
part of https://github.com/huggingface/diffusers/pull/8796

you can use this to test once this PR is fixed https://github.com/huggingface/diffusers/pull/8798

```python
from diffusers import LavenderFlowPipeline, LavenderFlowTransformer2DModel
import torch

repo = "AuraDiffusion/auradiffusion-v0.1a0"
dtype = torch.float16

transformer = LavenderFlowTransformer2DModel.from_pretrained(
    repo,
    revision="refs/pr/1",
    torch_dtype=dtype,
    subfolder="transformer"
    ) 

pipeline = LavenderFlowPipeline.from_pretrained(
	"AuraDiffusion/auradiffusion-v0.1a0", 
	revision="refs/pr/1",
    transformer=transformer,
	torch_dtype=torch.float16
).to("cuda")


images = pipeline(
    prompt="a cute cat with tiger like looks",
    height=512,
    width=512,
    num_inference_steps=50, 
    generator=torch.Generator().manual_seed(666),
    guidance_scale=3.5,
).images[0]
images.save("yiyi_test_4_out.png")
```

### CODE REVIEW

File: src/diffusers/pipelines/lavender_flow/pipeline_lavender_flow.py [hunk 1]
```diff
@@ -428,13 +429,14 @@ def __call__(
             prompt_embeds = torch.cat([negative_prompt_embeds, prompt_embeds], dim=0)
 
         # 4. Prepare timesteps
+
+        sigmas = np.linspace(1.0, 1 / num_inference_steps, num_inference_steps)
```
Comment (yiyixuxu) [inline src/diffusers/pipelines/lavender_flow/pipeline_lavender_flow.py:433]: I added this just to get 1:1 with the original implementation, I don't think it's needed here 
cc @cloneofsimo 

without this line
yiyi_test_4_out

with this line
yiyi_test_4_out_yiyi

File: src/diffusers/pipelines/lavender_flow/pipeline_lavender_flow.py [hunk 2]
```diff
@@ -448,20 +450,15 @@ def __call__(
...
+            for i, t in enumerate(timesteps):
                 # expand the latents if we are doing classifier free guidance
                 latent_model_input = torch.cat([latents] * 2) if do_classifier_free_guidance else latents
                 # broadcast to batch dimension in a way that's compatible with ONNX/Core ML
-                t = t / num_inference_steps
                 timestep = (
-                    torch.tensor([t]).expand(latent_model_input.shape[0]).to(latents.device, dtype=latents.dtype)
+                    torch.tensor([t / 1000])
+                    .expand(latent_model_input.shape[0])
+                    .to(latents.device, dtype=latents.dtype)
```
Comment (sayakpaul) [inline src/diffusers/pipelines/lavender_flow/pipeline_lavender_flow.py:461]: We could unpack this code a bit and perhaps add a comment on why we're dividing by 1000. This is mainly because we're moving away from our normal pipeline implementations a bit which our users are used to reading and referring to.

**Review by sayakpaul — APPROVED** [review]
Thanks. Just a single comment.

File: src/diffusers/pipelines/lavender_flow/pipeline_lavender_flow.py (hunk 2 above)
Comment (yiyixuxu) [reply src/diffusers/pipelines/lavender_flow/pipeline_lavender_flow.py]: done! feel free to merge into your PR now

**Comment by sayakpaul** [comment]
Thanks a lot for your help, Yiyi!
